github-to-slack/functions/index.js

```diff
@@ -33,26 +33,26 @@ exports.githubWebhook = functions.https.onRequest((req, res) => {
       // The JSON body is automatically parsed by Cloud Functions so we re-stringify it.
       .update(JSON.stringify(req.body, null, 0))
```

Comment:
Nit: I think we now have access to req.rawBody which will be more efficient/reliable than JSON stringifying rawBody